spec/support/features/in_person_helper.rb

```diff
@@ -77,6 +77,14 @@ def begin_in_person_proofing(_user = nil)
     click_button t('in_person_proofing.body.cta.button')
   end
 
+  def begin_in_person_proofing_with_opt_in_ipp_enabled_and_opting_in
```

Comment:
Small thought: maybe instead of saying 'opting in' say the path explicitly; IPP or Remote. It's obvious to us now with the context that we have that users are opting in to IPP, but that maybe will become more obscure in the future? IDK am I overthinking this?